src/adera/Machines/RCSController.h

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+#include <osp/Active/SysMachine.h>
+#include <Magnum/Math/Vector3.h>
+
+namespace adera::active::machines
+{
+
+class MachineRCSController;
+
+class SysMachineRCSController :
+    public osp::active::SysMachine<SysMachineRCSController, MachineRCSController>
+{
+public:
+    static inline std::string smc_name = "RCSController";
+
+    SysMachineRCSController(osp::active::ActiveScene &rScene);
+
+    void update_controls(osp::active::ActiveScene &rScene);
+
+    osp::active::Machine& instantiate(osp::active::ActiveEnt ent) override;
+
```

Comment:
Copyright

Reply:
Confused why github put this so far down the file. I meant it for the top of the file.